trade-finance-manager-ui/styles/table.scss

```diff
@@ -160,6 +160,7 @@ table.premium-payments-table {
 
 table.utilisation-report-reconciliation-table {
   table-layout: fixed;
+  width: calc(100% - 1px);
```

Comment:
Is it possible to reduce one of the columns by 1 pixel instead of creating a "negative" table width?

Reply:
I'm not sure this creates the negative table width. This just sets the table width to slightly less than 100% of its outer container and as the columns themselves have percentage-based widths they'll distribute their widths according to these within the table's slightly reduced width.

I also feel like reducing the width of the nth column would also be harder to understand vs applying this on the table element itself

Reply:
Sorry I meant like "less than 100% width" rather than "negative" table width, my fault for poor description 😄 

I know this is all minor, but feels strange to _not_ have the table width as 100% - it sounds like the root cause is one of the table's child elements/columns are making the overall table width more than 100%? If you temporary delete one of the columns and remove the new `calc(100% - 1px);` style, is the issue still present?

Am concerned that this issue could arise in the future with the "-1px" rule, if something is pushing the table out to be more than 1px.

Reply:
In the situation you describe, the issue of the scrollbar being shown is unfortunately still present (see screenshot at the bottom of this comment).

The issue is due to the browsers rounding when converting the percentage-based widths on each of the columns to their pixel values. The widths are converted to pixels with two decimal places meaning small rounding errors are present. Such errors should always be less than 1px in total (0.02px error observed in this case) and if not for some strange reason the scrollbar will be shown anyway (like in the screenshot below) so the page won't break.


![Screenshot 2024-09-09 110137](https://github.com/user-attachments/assets/3b84bd6d-f583-4307-a19b-2a86b6e92a34)

Reply:
Thanks @sw-mattw I think in the future we might need to refactor the table styles and/or potentially (with design) input maybe shortern or remove a column or two to help avoid such issues.

Happy for this to be merged in the meantime since it fixes an immediate issue (FN-2069) 👍